package.json

```diff
@@ -52,6 +52,7 @@
     "start": "yarn build && concurrently --kill-others -n Server,Rollup 'yarn serve' 'rollup -c -w'",
     "pretest": "yarn build:style",
     "test": "jest",
+    "watch": "jest --watch",
```

Comment:
You can already run `yarn test --watch` so I don't think we need another command. The main reason I want to not add this is since we have all the other Vega packages and I want to move towards consistency with fewer commands and setup requirements.